api/src/pcapi/core/external/attributes/api.py

```diff
@@ -139,7 +140,8 @@ def get_pro_attributes(email: str) -> models.ProAttributes:
     attributes = {}
 
     user = (
-        users_repository.find_pro_user_by_email_query(email)
+        # Also fetch NON_ATTACHED_PRO so that user is found, user id first name and last name are filled
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏
```suggestion
        # Also fetch NON_ATTACHED_PRO so that if user is found, user id first name and last name are filled

Reply:
Je voulais dire que ça permet de trouver l'utilisateur, mais comme je suis pressé, ça va plus vite de changer que négocier :smile: